scrapinghub/client/utils.py

```diff
@@ -133,35 +125,33 @@ def list(self, *args, **kwargs):
         return list(self.iter(*args, **kwargs))
 
 
-class _MappingProxy(_ProxyBase):
-
-    def __init__(self, cls, client, key):
-        super(_MappingProxy, self).__init__(cls, client, key)
+class _MappingProxy(_Proxy):
 
-        # cls_methods methods will be imported from origin class
-        origin_cls_methods = ['__iter__', '__len__', '__str__', '__repr__',
-                              '__getitem__', '__setitem__', '__delitem__']
+    def count(self):
+        return len(next(self._origin.apiget()))
```

Comment:
I added it only for consistency with other classes, `count` is almost everywhere. Dropping it?